### DISCUSSION TITLE
Display an error message if webpack build failed

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5658] [Feature Requests]
If webpack build results in an error, or some JS error occurs even before the page is rendered, the user would just see a blank white page, which is not helpful at all (though, there usually is a full error message in the console)

In cases like this, Specify should at the very least say that a JS-related error has occurred, and maybe even display the error message, or instruct the user to open the console.

**Comment by grantfitzsimmons** [comment]
This would be great! It would be especially nice if the error reported in the `specify7` container would be shown to the user.
